src/alfasim_sdk/_internal/alfacase/alfacase_to_case.py

```diff
@@ -137,8 +125,25 @@ def get_instance_loader(*, class_: type) -> Callable:
     return partial(load_instance, class_=class_)
 
 
+@lru_cache(maxsize=None)
+def get_dict_of_loader(*, class_: type) -> Callable:
+    """
+    Return a load  instance function pre-populate with the class_. @ramon change the docstring
+    """
+
+    def _get_dict_of_loader(alfacase_content: DescriptionDocument, class_: Type[T]) -> T:
```

Comment:
Rename `_get_dict_of_loader` -> `load_dict_of_instance` an move it to global scope.